.eslintrc

```diff
@@ -1,6 +1,9 @@
----
-extends:
-	- formidable/configurations/es6-node
-rules:
-  func-style: off
-  arrow-parens: off
+{
```

Comment:
NIT: I prefer YAML and the alignment is off in JSON here. (And it was off in L3 in original yaml)